egs/wsj/s5/steps/nnet3/multilingual/get_egs.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
```

Comment:
@danpovey 
I added steps/nnet3/multilingual/get_egs.sh, which generates egs..scp, outputs..scp, weights..scp for multilingual training using python file.
The details of egs..scp generation described in comments.
If you agree with whole pipeline for generating egs generation , I will start next steps, by doing following changes:
1. Modify nnet3-copy-egs to accept output and weight
2. add steps/nnet3/train_raw.py and modify it to be compatible with multilingual training setup.
3. modify all local/nnet3/run_* w.r.t new pipeline.